api/common/metadata/OAuthLoginResult.ts

```diff
@@ -0,0 +1,8 @@
+export interface OAuthResult {
```

Comment:
What is this interface based on? With Auth0 I see the following object associated with a logged-in user:

```
{
    "nickname": "",
    "name": "",
    "picture": "",
    "updated_at": "",
    "email": "",
    "email_verified": "",
    "sub": ""
}
```